modules/accredited_representative_portal/spec/requests/accredited_representative_portal/v0/power_of_attorney_request_decisions_spec.rb

```diff
@@ -27,15 +27,19 @@
   let!(:vso) { create(:organization, poa: poa_code, can_accept_digital_poa_requests: true) }
   let!(:other_vso) { create(:organization, poa: other_poa_code, can_accept_digital_poa_requests: true) }
 
-  let!(:poa_request) { create(:power_of_attorney_request, poa_code:) }
+  let!(:poa_request) do
+    create(:power_of_attorney_request, poa_code:,
+                                       accredited_individual_registration_number: '999999999999')
+  end
   let!(:other_poa_request) { create(:power_of_attorney_request, poa_code: other_poa_code) }
 
   let(:time) { '2024-12-21T04:45:37.458Z' }
 
   before do
+    allow_any_instance_of(Auth::ClientCredentials::Service).to receive(:get_token).and_return('fake_access_token')
     Flipper.enable(:accredited_representative_portal_pilot)
```

Comment:
Please mock accredited_representative_portal_pilot  flippers: `allow(Flipper).to receive(:enabled?).with(: accredited_representative_portal_pilot, instance_of(User)).and_return(true)`

Reply:
Please remove  Flipper.enable(:accredited_representative_portal_pilot)